components/cam/src/physics/cam/micro_p3.F90

```diff
@@ -1324,7 +1188,7 @@ SUBROUTINE p3_main(qc,nc,qr,nr,th_old,th,qv_old,qv,dt,qitot,qirim,nitot,birim,ss
 
           !................................................................
           ! saturation adjustment to get initial cloud water
-
+          ! AaronDonahue: Do we need to get rid of this too?
```

Comment:
Yes, I don't think qccon should be calculated anywhere here. Note that this assumes macrophysics occurs before microphysics... but I think that's appropriate.

Reply:
Excellent, I'll purge all remaining references to qccon